api/src/main/java/org/pmiops/workbench/db/model/DataSet.java

```diff
@@ -180,16 +180,19 @@ public void setValues(List<DataSetValues> values) {
   }
 
   @Column(name = "prePackagedConceptSet")
-  public short getPrePackagedConceptSet() {
+  public Short getPrePackagedConceptSet() {
     return prePackagedConceptSet;
   }
 
-  public void setPrePackagedConceptSet(short prePackagedConceptSet) {
+  public void setPrePackagedConceptSet(Short prePackagedConceptSet) {
     this.prePackagedConceptSet = prePackagedConceptSet;
   }
 
   @Transient
   public PrePackagedConceptSetEnum getPrePackagedConceptSetEnum() {
+    if (this.prePackagedConceptSet == null) {
```

Comment:
We don't usually want `get` methods to mutate state, unless it's a lazy evaluation or other optimization. I'd probably move this check to the `set` method.

Reply:
I actually do not need this check removing it